src/main/java/org/myrobotlab/service/OpenCV.java

```diff
@@ -513,7 +545,7 @@ public static IplImage cropImage(IplImage img, CvRect rect) {
 
   boolean undockDisplay = false;
 
-  transient Thread videoThread = null;
+  transient Thread videoThreadx = null;

```

Comment:
odd variable name.. why videoThread"x" ?

Reply:
identified an unused variable ... sadly when you initialize to null, the ide linter isn't smart enough to tell that its not used ... removed this var since it is unused...